src/Cli/Microsoft.DotNet.Cli.Utils/RuntimeEnvironment.cs

```diff
@@ -65,35 +65,15 @@ private static string GetOSVersion()
                 case Platform.Darwin:
                     return Environment.OSVersion.Version.ToString(2);
                 case Platform.FreeBSD:
-                    return GetFreeBSDVersion() ?? string.Empty;
-                case Platform.Solaris:
-                    // RuntimeInformation.OSDescription example on Solaris 11.3:      SunOS 5.11 11.3
-                    // we only need the major version; 11
-                    return RuntimeInformation.OSDescription.Split(' ')[2].Split('.')[0];
```

Comment:
Have you ensured that Solaris returns the correct major version for Environment.OSVersion.Version?

Reply:
Do we have any tests in this area?